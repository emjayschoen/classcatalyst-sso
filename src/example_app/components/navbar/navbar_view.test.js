import React from 'react';
import { shallow } from 'enzyme';
import NavbarView from './navbar_view';

describe('<NavbarView />', () => {
  let subject;

  describe('signed in', () => {
    beforeEach(() => {
      subject = shallow(<NavbarView signedIn />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });

  describe('signed out', () => {
    beforeEach(() => {
      subject = shallow(<NavbarView signedIn={false} />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });
});
