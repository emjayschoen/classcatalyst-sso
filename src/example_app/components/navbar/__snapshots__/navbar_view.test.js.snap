// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarView /> signed in renders correctly 1`] = `
<styled.div>
  <h3>
    NAV BAR
  </h3>
  <p>
    You are currently
     signed in!
  </p>
</styled.div>
`;

exports[`<NavbarView /> signed out renders correctly 1`] = `
<styled.div>
  <h3>
    NAV BAR
  </h3>
  <p>
    You are currently
     signed out!
  </p>
</styled.div>
`;
